app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greeting = Greeting.all.sample
+    render json: { greeting: @greeting }
```

Comment:
- [ ] **[OPTIONAL]** I don't think it's a good idea to return an entire record as we have in the database in an API response. I think it would be nice to return only the `message` attribute.

  ![utu](https://user-images.githubusercontent.com/36542986/187187456-2a28dc4c-8d03-452b-95b5-d555810402db.jpg)